airbyte-integrations/connectors/destination-clickhouse-strict-encrypt/src/test/resources/expected_spec.json

```diff
@@ -46,11 +46,79 @@
         "airbyte_secret": true,
         "order": 4
       },
+      "ssl_mode": {
+        "title": "SSL modes",
+        "description": "SSL connection modes. \n <b>disable</b> - Chose this mode to disable encryption of communication between Airbyte and destination database\n <b>allow</b> - Chose this mode to enable encryption only when required by the source database\n <b>prefer</b> - Chose this mode to allow unencrypted connection only if the source database does not support encryption\n <b>require</b> - Chose this mode to always require encryption. If the source database server does not support encryption, connection will fail  \n See more information - <a href=\"https://jdbc.postgresql.org/documentation/head/ssl-client.html\"> in the docs</a>.",
+        "type": "object",
+        "order": 5,
+        "oneOf": [
+          {
+            "title": "disable",
```

Comment:
Seems like according to the clickhouse documentation, the possible values for the ssl_mode are "none" or "strict" or "relaxed", we probably only need to keep "none and "strict" here to keep the same functionality. 

Also slight typo "Chose" should be "Choose" here :)